pom.xml

```diff
@@ -56,11 +56,10 @@
     </dependencyManagement>
 
     <dependencies>
-        <!-- Logging -->
         <dependency>
-            <groupId>io.gravitee.node</groupId>
-            <artifactId>gravitee-node-api</artifactId>
-            <version>${gravitee-node.version}</version>
+            <groupId>io.gravitee.gateway</groupId>
+            <artifactId>gravitee-gateway-api</artifactId>
+            <version>${gravitee-gateway-api.version}</version>
```

Comment:
gravitee-gateway-api already has gravitee-common in its dependencies. It will create a cycle don't you think?

Reply:
Yes you're right! I rolled back the upgrade of gravitee-node for now